test/e2e/helm/kuma_helm_upgrade.go

```diff
@@ -69,7 +69,6 @@ func UpgradingWithHelmChart() {
 			// then CRD is upgraded
 			Expect(err).ToNot(HaveOccurred())
 			Expect(out).To(ContainSubstring("AllowWithShadowDeny"))
-			// remove this when+then after initialChartVersion is changed to 2.1.x or later
```

Comment:
I think this whole test is not really doing anything useful now.

Reply:
It seems like it upgrades and check MTP default value. I think we can remove it now and in case we need it just write a new one.